mvz/methods/shared.py

```diff
@@ -10,6 +10,11 @@
 # to the left.
 min_value_x = 100
 
+Frame = Tuple[float, float, float, float, float]
+Frames = List[Frame]
+FrameSpecOutput = Union[List[const.BoundingBox],
+                        Frames]
```

Comment:
Yeah, it's a little repetitive.  We could also just change everything to pass around `floats` too, if you like that better.